WordPressAuthenticator/Signin/LoginPrologueViewController.swift

```diff
@@ -138,12 +143,14 @@ class LoginPrologueViewController: LoginViewController {
         }
 
         vc.googleTapped = { [weak self] in
+            guard let self = self else { return }
```

Comment:
Same here - `return` should be on a separate line.

Reply:
Changed.